scio-examples/src/main/scala/com/spotify/scio/examples/extra/ParquetExample.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2019 Spotify AB.
```

Comment:
copyright date needs to be updated! you can set it dynamically in your IDE like this: https://spotify.github.io/scio/dev/Style-Guide.html#intellij-idea